// connect-auth/examples/keys_file.js
exports.bitbucketConsumerKey= "";
exports.bitbucketConsumerSecret= "";
exports.bitbucketCallbackAddress= "http://yourtesthost.com/auth/bitbucket_callback";
exports.fbId= "";
exports.fbSecret= "";
exports.fbCallbackAddress= "http://yourtesthost.com/auth/facebook_callback";
exports.getGlueId= "";
exports.getGlueSecret= "";
exports.getGlueCallbackAddress= "http://yourtesthost.com/auth/getglue_callback";
exports.ghId= "50d5b8dcdbb4d01a21e2";
exports.ghSecret= "f830d88368d667a836da8e56bbe91a04f3d965ba";
exports.ghCallbackAddress= "http://yourtesthost.com/auth/github_callback";
exports.twitterConsumerKey= "";
exports.twitterConsumerSecret= "";
exports.yahooConsumerKey= "";
exports.yahooConsumerSecret= "";
exports.yahooCallbackAddress= "http://yourtesthost.com/auth/yahoo_callback";
exports.googleConsumerKey= "";
exports.googleConsumerSecret= "";
exports.googleCallbackAddress= "http://yourtesthost.com/auth/google_callback";
exports.google2Id= "";
exports.google2Secret= "";
exports.google2CallbackAddress= "http://yourtesthost.com/oauth2callback";
exports.foursquareId= "";
exports.foursquareSecret= "";
exports.foursquareCallbackAddress= "http://yourtesthost.com/auth/foursquare_callback";
exports.janrainApiKey= "";
exports.janrainAppDomain= "yourrpxnowsubdomain";
exports.janrainCallbackUrl= "http://localhost/auth/janrain_callback";
exports.openIdCallback= "http://yourtesthost.com/verify";
exports.linkedinConsumerKey= "";
exports.linkedinConsumerSecret= "";
exports.linkedinCallback= "http://localhost/auth/linkedin_callback";
exports.personaAudience="yourtesthost.com";
exports.yammerConsumerKey= "";
exports.yammerConsumerSecret= "";
exports.yammerCallback= "http://localhost/auth/yammer_callback";
exports.skyrockConsumerKey= "";
exports.skyrockConsumerSecret= "";
exports.skyrockCallbackAddress= "http://local.host/auth/skyrock_callback";
